server/middleware/one-time-code-auth/setupStartAgain.ts

```diff
@@ -0,0 +1,9 @@
+import express, { Router } from 'express'
+
+export default function setupStartAgain(): Router {
+  const router = express.Router()
+
+  router.get('/start-again', (req, res) => res.render('pages/one-time-code-auth/startAgain'))
```

Comment:
A new page is triggered after running out of attempts at the old code. For some reason.